scripts/v.unpack/v.unpack.py

```diff
@@ -77,7 +77,7 @@ def main():
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
         data_name = tar.getnames()[0]
-    except:
+    except (tarfile.ReadError, IndexError):
```

Comment:
There is no tarfile exception mentioned for `getnames` indocumentation, the error would probably happen during openening it, not here.